src/network/rs225/client/handler/ClientCheatHandler.ts

```diff
@@ -40,13 +40,13 @@ export default class ClientCheatHandler extends MessageHandler<ClientCheat> {
         }
 
         if (player.staffModLevel >= 2) {
-            player.addSessionLog(LoggerEventType.MODERATOR, 'Ran cheat', cheat);    
+            player.addSessionLog(LoggerEventType.MODERATOR, 'Ran cheat', cheat);
         }
 
         if (!Environment.NODE_PRODUCTION && player.staffModLevel >= 3) {
             // developer commands
 
-            if (cmd[0] === '~') {
+            if (cmd[0] === Environment.NODE_DEBUGPROC_CHAR) {
```

Comment:
```suggestion
            if (cmd[0] === Environment.NODE_DEBUGPROC_CHAR || '~') {
```
In case the env variable is commented out/undefined have this default fallback

Reply:
nvrm I think its handled in Environment.ts

Reply:
yeah we use `tryParseString` I guess what could go wrong is that NODE_DEBUGPROC_CHAR is an empty string I'll add this just incase someone does that